the_list.html

```diff
@@ -258,6 +274,16 @@ <h2>The List</h2>
             <td>15+</td>
             <td><div>Psychological Horror, Music to induce Panic, Discussion of Abuse,</div><div>Abstract Homophobia, Abstract Transphobia.</div></td>
         </tr>
+        <tr>
+            <td>Fire Emblem (The Blazing Blade)</td>
+            <td>
+                <a href="https://en.wikipedia.org/wiki/Fire_Emblem_(video_game)">Nintendo Game Boy Advance</a>
+            </td>
+            <td>Strategy, RPG, Fantasy</td>
+            <td>Limstella (Agender, Minor villain)</td>
+            <td>7+</td>
+            <td>Fantasy Violence, Domestic Abuse</td>
```

Comment:
Less important than the others. There is some messed up stuff in this game, but I don't know all of which is worthy of a tag.

There is a support which references slavery, Karel is nuts, and Priscilla's got some creepy stuff. Still all of this is regulated to supports. I feel like there is a character in there that has ptsd.

Fire Emblem is weird like that. E rating but some very unsettling implications.

That said this is probably fine as is and I'm mostly leaving it up to you for what tags to add. 

I'm still not certain on Limstella.